CHANGELOG.md

```diff
@@ -9,10 +9,11 @@ The latest version is considered WIP and it is a subject of change. All other
 versions are frozen. To freeze current version all files are copied to a new
 version directory, and  then changes are introduced.
 
-## [v5.1.0] - 2020-01-21
+## [v5.1.0] - Unreleased
```

Comment:
I got a feeling this version is already released so this change should go to the new version of k8scloudconfig

Reply:
@MarcelMue can you confirm -- should I add a 5.2.0 version or is 5.1.0 still waiting?

Reply:
kvm is using `5.0.0` https://github.com/giantswarm/kvm-operator/blob/2fd4e6e377b279837f89b915b465d579510c4afb/service/controller/cloudconfig/master_template.go#L8
aws is using `5.0.0` https://github.com/giantswarm/aws-operator/blob/master/service/controller/internal/cloudconfig/tccp.go#L11

BUT azure uses `5.1.0` https://github.com/giantswarm/azure-operator/blob/b23abfe0195bdfc7ee99d533fdc39cd186d0d479/service/controller/v13/cloudconfig/master_template.go#L9

So would be best to ask @giantswarm/team-celestial why they didn't create a new `k8scc` version when they created their azure release :P

Reply:
Added v_5_2_0

Reply:
:see_no_evil: Need to add that as part of release checklist. Totally forgot.